src/12-tickets/tickets.apib

```diff
@@ -46,6 +46,12 @@ Get a list of tickets.
                     + customer (object)
                         + type: `company` (string)
                         + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+                + project (object, nullable)
```

Comment:
Q: Should we mention these will only be included if the user's still using the old projects module?